tests/unit/metrics/test_metric_targets.py

```diff
@@ -261,14 +263,37 @@ def test_convert_metrics_to_pretty_json_with_empty_labels(self):
 
         assert exposed_json == expected_json
 
-    def test_add_timestamp_adds_meta_subfield_with_current_timestamp(self):
+    def test_add_meta_information_adds_meta_subfield_with_timestamp_and_versions(self):
         metric_json = {"pipeline": "not important here"}
-        metric_json = self.target._add_timestamp(metric_json)
+        metric_json = self.target._add_meta_information(metric_json)
         assert "meta" in metric_json
         assert "timestamp" in metric_json["meta"].keys()
         assert isinstance(
             datetime.strptime(metric_json["meta"]["timestamp"], "%Y-%m-%dT%H:%M:%S.%f"), datetime
         )
+        expected_versions = {
+            "logprep": get_versions()["version"],
+            "config": self.logprep_config.get("version"),
+        }
+        assert metric_json["meta"]["version"] == expected_versions
+
+    def test_add_meta_information_adds_unset_for_config_version_if_not_found(self):
+        logger = logging.getLogger("test-file-metric-logger")
+        logprep_config = {"other_fields": "are_unimportant_for_test"}
```

Comment:
you might want to add a comment here that explains the fact that at this point, a version could have been set and that this is essential for this test.